dashboards/mintel/experimental/kubernetes-cluster-health-dashboard.json

```diff
@@ -382,31 +383,21 @@
       "tableColumn": "",
       "targets": [
         {
-          "expr": "1 + (absent(count(up{job=~\"kubelet|etcd|kube-state-metrics|node-exporter\"} == 0)) or vector(0))",
+          "expr": "sum(kube_node_status_condition{condition!=\"Ready\", status=\"true\"} )",
```

Comment:
should we `SUM by (node)`  so that if one node is bad ( with many conditions) we will know is just one node 

and maybe rename to Nodes with Bad Conditions

Reply:
Yeah not a bad shout

Reply:
actually something like 

`sum(sum by (node) (kube_node_status_condition{condition!=\"Ready\", status=\"true\"} ) bool > 0)`

Reply:
bool didnt work, I was recieving errors.  
This works however:
`count(sum by (node)(kube_node_status_condition{condition!="Ready", status="true"}) > 0) OR on() vector(0)`